docs/guides/embeds.md

```diff
@@ -0,0 +1,83 @@
+# How to Embed the Video.js player
```

Comment:
after I added this page I remember that we have the `setup.md` file. This page can probably stay as is but `setup.md` probably should be updated a bit as well and point to this page.

Reply:
Linked from `setup.md` now.